src/relay/transforms/to_basic_block_normal_form.cc

```diff
@@ -51,8 +51,11 @@ Expr ToBasicBlockNormalFormAux(const Expr& e) {
 IRModule ToBasicBlockNormalForm(const IRModule& mod) {
   DLOG(INFO) << "ToBBlock:" << std::endl << mod;
 
+  // Efficient copy.
```

Comment:
Can we make a more direct comment here? Maybe the "Efficient copy." won't help other developers more because the code is just a copying. Can we change it to something like "Efficient copy for avoiding change the original IRModule."?

Reply:
Alternatively we could say "shallow copy"

Reply:
what about now?

Reply:
I don't think we need such explanation here actually lol. Let's just keep it brief and say it is a shallow copy :-)